### ISSUE TITLE
adb' is not recognized  in  vs code

### ISSUE THREAD

**Issue by fahadfarooqstar** [aakash-pamnani/ADB-QR#15]
ADB QR: 'adb' is not recognized as an internal or external command, operable program or batch file.

**Comment by biefchtekz** [comment]
Same problem, any fixes?

**Comment by fahadfarooqstar** [comment]
Yeah you can do one thing add the path of ADB in your environment variables as well as system variables after adding the path you will have to refresh the system and restart vs code to make changes in the ADB than search adb add in the the search pallette of vs code to run the adb and turn on your wireless USB debugging and than write your mobile cose to adb as shown that,s it

**Comment by S-N-O-R-L-A-X** [comment]
The same. The adb has been in the env path.

**Comment by S-N-O-R-L-A-X** [comment]
![Image](https://github.com/user-attachments/assets/9435ecd0-7551-4e8e-a99b-6e5096b1d827)

![Image](https://github.com/user-attachments/assets/bf759c13-776f-49fe-b219-29be57d2f4ec)

**Comment by aakash-pamnani** [comment]
@S-N-O-R-L-A-X try running `adb` command in vscode terminal, and check if it works?

**Comment by adriansasu** [comment]
Same problem here
